security/compliance_operator/compliance-operator-troubleshooting.adoc

```diff
@@ -0,0 +1,27 @@
+[id="compliance-operator-troubleshooting"]
+= Troubleshooting the Compliance Operator
+include::modules/common-attributes.adoc[]
+:context: compliance-troubleshooting
+
+toc::[]
+
+This section describes how to troubleshoot the Compliance Operator. The information can be useful either to diagnose a problem or provide information in a bug report. Some general tips:
+
+* The Compliance Operator emits Kubernetes events when something important happens. You can either view all events in the cluster using the command `oc get events -nopenshift-compliance` or events for an object, such as a scan using the command `oc describe compliancescan/$scan`.
```

Comment:
for`oc describe compliancescan/$scan`, is it better to use `oc describe compliancescan/<scan_name>`?